grails-app/controllers/au/org/ala/collectory/DataResourceController.groovy

```diff
@@ -240,7 +245,8 @@ class DataResourceController extends ProviderGroupController {
         }
     }
 
-    def updateConsumers = {
+    @PermissionRequired(roles=['ROLE_EDITOR', 'ROLE_ADMIN'])
```

Comment:
Remove this annotation. It is already on the controller class.

Reply:
removed